### PR TITLE
Support floating labels on `.form-control-plaintext`

### PR DESCRIPTION
**Pull request by SPodjasek**
I've made a tiny fix that will allow usage of floating labels with `.form-control-plaintext` elements.

Before:
screenshot-20210119_153457

After:
screenshot-20210119_153601

I'm not sure that this requires a dedicated section in the documentation, but as part of PoC, I've made one, so it's included. Feel free to wipe it out.

Preview: https://deploy-preview-32840--twbs-bootstrap.netlify.app/docs/5.0/forms/floating-labels/#readonly-plain-text

### CODE REVIEW

**Comment by mdo** [comment]
Empty input looks broken, maybe we don't include it?

**Comment by SPodjasek** [comment]
IMHO due to the lack of borders, it will look better when label is shown like there's some value in input...

image

**Comment by SPodjasek** [comment]
@mdo @XhmikosR What do you think about this?

Example

**Comment by SPodjasek** [comment]
Any chances this will land in next minor release?

**Comment by XhmikosR** [comment]
@mdo should this be in 5.2.0 or 5.1.2?

**Comment by SPodjasek** [comment]
@XhmikosR sorry I've dropped your commit with `--force-push`, haven't seen it

File: site/content/docs/5.1/forms/floating-labels.md [hunk 1]
```diff
@@ -75,6 +75,21 @@ Other than `.form-control`, floating labels are only available on `.form-select`
 </div>
 {{< /example >}}
 
+## Readonly plain text
```
Comment (mdo) [inline site/content/docs/5.1/forms/floating-labels.md:78]:
```suggestion
## Readonly plaintext
```

File: site/content/docs/5.1/forms/floating-labels.md [hunk 2]
```diff
@@ -75,6 +75,21 @@ Other than `.form-control`, floating labels are only available on `.form-select`
...
+
+You can also use floating labels with the `.form-control-plaintext` class to remove the default form field styling and preserve the correct margin and padding.
```
Comment (mdo) [inline site/content/docs/5.1/forms/floating-labels.md:80]:
```suggestion
Floating labels also support `.form-control-plaintext`, which can be helpful for toggling from an editable `<input>` to a plaintext value without affecting the page layout.
```

**Review by mdo — APPROVED** [review]

**Pull request merged**